Packages/vcs/Lib/VTKAnimate.py

```diff
@@ -122,8 +122,6 @@ def __init__(self,vcs_self):
         self.renderers = []
         self.last_size = None
         self.modified_listener = None
-        import atexit
-        atexit.register(self.close)
```

Comment:
As long as close is called explicitly, it should behave just as before. The reason `__del__` wasn't getting called is likely due to reference leaks -- if there are lingering references due to cycles or `atexit` registrations, `__del__` will never be called since the garbage collector will never be notified to clean the objects up.

+1 for using platform temporary files instead of the home directory.